pkg/compose/down.go

```diff
@@ -53,6 +53,10 @@ func (s *composeService) down(ctx context.Context, projectName string, options a
 	project := options.Project
 	if project == nil {
 		project, err = s.getProjectWithResources(ctx, containers, projectName)
+		if api.IsNotFoundError(err) {
```

Comment:
Not sure that `s.getProjectWithResources` work for this verification.
From what I see, it returns in the first occurrence of a not found resource. 
So if you still have another type of resource to remove (let's say `volumes`) it would be ignored.

https://github.com/docker/compose/blob/v2/pkg/compose/down.go#L239-L258

Reply:
indeed, but that's another issue :P
with no container, we just quit with error not found

Reply:
I had in mind many time to refactor `projectFromName` into a set of `actualServices`+`actualVolumes`+`actualNetworks`  so that we can better manage this. But this is a way bigger change